src/api/procedures/modifyMultiSig.ts

```diff
@@ -0,0 +1,101 @@
+import { MultiSig } from '~/api/entities/MultiSig';
+import { PolymeshError, Procedure } from '~/internal';
+import { ErrorCode, Signer, TxTags } from '~/types';
+import { signerToSignatory, signerToString, stringToAccountId } from '~/utils/conversion';
+
+/**
+ * @hidden
+ */
+export interface ModifyMultiSigParams {
+  multiSig: MultiSig;
+  signers: Signer[];
+}
+
+/**
+ * @hidden
+ */
+function calculateSignerDelta(current: Signer[], target: Signer[]): [Signer[], Signer[]] {
+  const currentSet = new Set(current.map(signerToString));
+  const targetSet = new Set(target.map(signerToString));
+
+  const newSigners = new Set([...target].filter(s => !currentSet.has(signerToString(s))));
+  const removedSigners = new Set([...current].filter(s => !targetSet.has(signerToString(s))));
+
+  return [Array.from(newSigners), Array.from(removedSigners)];
+}
+
+/**
+ * @hidden
+ */
+export async function prepareModifyMultiSig(
+  this: Procedure<ModifyMultiSigParams, void>,
+  args: ModifyMultiSigParams
+): Promise<void> {
+  // adds or removes signers and can change number of signatures required.
```

Comment:
```suggestion
  // adds or removes signers
```

Also, shouldn't these comments go at the top of the function?